src/main/java/io/boomerang/service/crud/WorkflowScheduleServiceImpl.java

```diff
@@ -205,6 +205,9 @@ public WorkflowSchedule createSchedule(final WorkflowSchedule schedule) {
           Boolean enableJob = false;
           if (WorkflowScheduleStatus.active.equals(schedule.getStatus()) && wfEntity.getTriggers().getScheduler().getEnable()) {
             scheduleEntity.setStatus(WorkflowScheduleStatus.trigger_disabled);
+            if (WorkflowScheduleType.runOnce.equals(scheduleEntity.getType()){
+              scheduleEntity.setStatus(WorkflowScheduleStatus.active);
+            }
```

Comment:
Hey @florentinvintila can you clarify, do we want all schedules to be created active or just runonce schedules?

On line 207 we set the status to trigger_disabled... but the wrapping condition on 206 is that the trigger is enabled. Should we be removing line 207 and just set to active for all types?